### PR TITLE
Update UG

### LINKED ISSUE #100: Update UG

### PR DESCRIPTION
**Pull request by marraknas**
Fixes #100

### CODE REVIEW

**Review by PeterHW963 — COMMENTED** [review]
LGTM

**Comment by ulricolo7** [comment]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ FindingbrUdders is a **desktop app for managing contacts and scheduling meetings
 ### Viewing help : `help` 💡
 
 Shows a message via a popup window explaining how to access the help page.
```
Comment (XinghuaJulia) [inline docs/UserGuide.md:85]: Is this line redundant with the new "help" description added?

**Review by XinghuaJulia — COMMENTED** [review]
LGTM! I like how more examples are added to further illustrate how to use the command :DDDDDDD

**Pull request merged**